### ISSUE TITLE
Implement description of approximate value of segment and corners to toolbar buttons?

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/quadrilateral#187]
We discussed this at a quad meeting a few weeks back and I wanted to bring it up for discussion or possible implementation.

### Objective

To align with the visual estimation abilities afforded by the line segments on the sides and pie segments on the corner guides - and to help with estimation tasks (area & perimeter) - should we considered listing the approximate segment lengths and angles of the current shape in the Details or Hint button?

### Visual/Model Definitions of Length/Angle

Currently, visual length segments are the same size as the grid currently. The length tolerance is 0.01 which is 1% of the play area or something similar to that? Angles are broken into 30-degree slices (3 slices = right angle, 6 slices = semicircle/180 degrees, etc). The angle tolerance is about 1-2 degrees (converted from 0.01 radians). These do change quite a bit with the tangible, however.

### Possible Complications or Design Obstacles

The tricky bit here is to determine what level of detail is appropriate. It'd be ideal if it wasn't so broad that hearing the angle values for a rhombus sounds like a square, etc but I am pretty sure this is not feasible. (for example, from the starting square, trying moving vertex A left, right, up down, and diagonally by 1 or two minor ticks. You'll see 4 shapes with very similar lengths/angles.

Describing length and angle do feel a bit different. Feels more okay to me to describe the length in segments rather than a measurement unit than describing the angle in pie segments instead of degrees... 

A couple of possibilities/examples:

### Possible Length Descriptions

For length: Quarter or tenth of segments? 
- e.g., "Side AB, about four and a {quarter/half/three quarters} segments" 
- or "Side AB, about four and {one tenth/(two/three/four) tenths/a half/(six/seven/eight/nine) tenths}

Drawing showing how the described region would need to differ from the actual length:
![SharedScreenshot](https://user-images.githubusercontent.com/47331395/183106406-7b78cb11-b021-442f-9e7a-47bf079ac85c.jpg)

### Possible Angle Descriptions

For angle: 10-degree resolution (= 3 bins per pie slice)? 
- e.g., "Corner A, about 100 degrees" 
- or "Corner A, about three and a third slices"

Drawing showing how the described region would need to differ from the actual angle. I think the same regions are appropriate for either approximate degree or slice description.

![SharedScreenshot2](https://user-images.githubusercontent.com/47331395/183106478-76c30449-9def-42a4-add9-17f657807c98.jpg)
![Angle Describe_CloseUp ](https://user-images.githubusercontent.com/47331395/183106496-8ec6032a-5b16-4308-9645-2d760c4e5497.jpg)

Assigning @terracoda and @emily-phet for discussion.

**Comment by emily-phet** [comment]
Given the young age group this sim reaches, this is quite tricky. My guess is that terms like "a third" might not be effective for the ~ under 8 crowd. I wish their was a plain-language term for "90 degree angle", so for the angle description it could be used as a benchmark (distance to/from 90 degree angle). I can't think of one. 

If this still being considered, let's discuss August 23rd (which I'm guessing will be after some interviews where we might get a renewed sense of how increasingly precise description might be useful, or not).

**Comment by terracoda** [comment]
“Right Angle” is the plain language term we are currently using.

**Comment by terracoda** [comment]
Of course, one could drop the comparison in the show values mode and just provide the value:
- Corner {{A}}, about {{A}} degrees 
- Corner {{B}}, about {{B}} degrees 
- Corner {{C}}, about {{C}} degrees
- Corner {{D}}, about {{D}} degrees

Maybe that's what you are proposing. We'll discuss more at design meeting.

**Comment by BLFiedler** [comment]
We didn't get back to this, but yeah, I suppose where this would work best was just a separate Voicing button altogether. I think from your interviews with the consultants, they were looking for a few more precise bits of information (e.g., grid values). This would manifest as just a list of values more or less.

Let's defer this idea until we get the other descriptions sorted out. We'll also need #181 sorted first. Also, as EM noted, language/interval choice here can be tricky when considering elementary to middle school.

**Comment by BLFiedler** [comment]
Agreed. I was going to comment along those lines that I think that will do well to accomplish this goal without getting overly quantitative.
